tools/internal_ci/linux/grpc_bazel_test_in_docker.sh

```diff
@@ -1,29 +0,0 @@
-#!/usr/bin/env bash
-# Copyright 2017 gRPC authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-# Test full Bazel
-#
-# NOTE: No empty lines should appear in this file before igncr is set!
-set -ex -o igncr || set -ex
-
-mkdir -p /var/local/git
-git clone /var/local/jenkins/grpc /var/local/git/grpc
-(cd /var/local/jenkins/grpc/ && git submodule foreach 'cd /var/local/git/grpc \
-&& git submodule update --init --reference /var/local/jenkins/grpc/${name} \
-${name}')
-cd /var/local/git/grpc
-
-bazel test //test/...
-bazel test //test/cpp/end2end:admin_services_end2end_test --define=grpc_no_xds=true
```

Comment:
Can you help to move this test to a correct place? This test itself is still valuable, as it validates that the given target works without any xDS integration.

Reply:
The only place I know of where we could add this is 
https://github.com/grpc/grpc/blob/c05ecb8a9e99df3ff7cf40c1a947a85b3b4a56c6/tools/internal_ci/linux/grpc_bazel_build_in_docker.sh#L48
(but note that that's a "build only" job, it doesn't run the test, just verifies it builds).
AFAIK Currently we don't have any test that does "bazel test" on linux (except the RBE jobs of course).

Please let me know what you wanna do.

Reply:
It seems putting `grpc_no_xds` mode on RBE is the most correct way? If you have cycles, can you help? Otherwise, feel free to assign to me.

Reply:
If so, you can do it by basically duplicating these files https://github.com/grpc/grpc/blob/master/tools/internal_ci/linux/grpc_bazel_rbe_opt.cfg
https://github.com/grpc/grpc/blob/f8a909e76fcd947949502832a7ab8e2cba2b8e27/tools/internal_ci/linux/grpc_bazel_on_foundry_opt.sh
(while setting --define=grpc_no_xds=true) and creating a new kokoro job for it.
the name of the new job can be e.g. `grpc_bazel_rbe_no_xds`. 
Since so far we weren't running grpc_bazel_test job at all, I think it's enough to add that job for master only.